### ISSUE TITLE
Restore the database

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-ops/localization#14]
I have to restore a database backup on localize.backdropcms.org. Before I changed a configuration today, I made a database backup, and now I need to switch back to the state before.

Unfortunately, it's not possible to upload my backup via the *Backup and Migrate" restore page. I get a ["Request Entity Too Large" error](https://github.com/backdrop-ops/localization/issues/16). The `.mysql` file is only 5MB, and PHP max file size is 64MB, so the latter shouldn't be the issue.

@jenlampton or @larsdesigns - I'd appreciate if you could check the server regarding the "Entity Too Large" issue (it applies also when I try to upload language files with more than 1MB).

However, for the moment I'd be happy with a quick restore of the .mysql file via phpMyAdmin or so. @jenlampton I'll send you the file via email.

**Comment by serundeputy** [comment]
@jenlampton @olafgrabienski I can't do this w/out server access; @larsdesigns care to allow me access to the server?

**Comment by larsdesigns** [comment]
@serundeputy I will check PHP's file upload settings later in the morning.

**Comment by larsdesigns** [comment]
@serundeputy if you find me on gitter, we can work through this together.

**Comment by larsdesigns** [comment]
@serundeputy 

NGINX looks okay.

```
larsdesigns@www1 /e/nginx> sudo grep -rni "client_max_body_size" *
[sudo] password for larsdesigns: 
conf.d/sanitize.backdropcms.org.conf:26:    client_max_body_size 100M;
conf.d/backdropcms.org.conf:25:    client_max_body_size 100M;
conf.d/sanitize.backdropcms.org.conf~:26:    client_max_body_size 100M;
```

**Comment by larsdesigns** [comment]
@serundeputy PHP looks like this: 

```
larsdesigns@www1 /etc> sudo php -i | grep "max"
apc.max_file_size => 1M => 1M
log_errors_max_len => 1024 => 1024
max_execution_time => 0 => 0
max_file_uploads => 20 => 20
max_input_nesting_level => 64 => 64
max_input_time => -1 => -1
max_input_vars => 10000 => 10000
post_max_size => 100M => 100M
upload_max_filesize => 64M => 64M
PHP Warning:  Unknown: It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. in Unknown on line 0
mysql.max_links => Unlimited => Unlimited
mysql.max_persistent => Unlimited => Unlimited
mysqli.max_links => Unlimited => Unlimited
mysqli.max_persistent => Unlimited => Unlimited
session.gc_maxlifetime => 1440 => 1440
```

**Comment by larsdesigns** [comment]
@serundeputy 

```
larsdesigns@www1 /etc> sudo grep -rni "post_max_size" php.ini 
676:post_max_size = 100M
larsdesigns@www1 /etc> sudo grep -rni "upload_max_filesize" php.ini 
804:upload_max_filesize = 64M
```

**Comment by larsdesigns** [comment]
@olafgrabienski, it sound like a database resource configuration problem rather than PHP. Can you provide the database dump to me. I will investigate the error and import the database from using "source".

**Comment by olafgrabienski** [comment]
@larsdesigns Sure, I'll send you the `.mysql` file. Can you give me your email address via a PM in Gitter?

(Going to be offline for a couple of hours.)

**Comment by larsdesigns** [comment]
@olafgrabienski: I backed up the existing localize database, dropped the localize database, recreated the localize database, and imported the data from the localize.backdropcms.org-2018-08-28T13-03-37-before-startover-releases.mysql file using the source command. There were no errors or warnings in the output. The import seemed successful fro the command line.

**Comment by larsdesigns** [comment]
I apologize for the delay.

**Comment by olafgrabienski** [comment]
@larsdesigns Thanks for importing the mysql data! I've just had a look at the site, and as far as I can see, the site configuration has been reset successfully.

As expected, the import solved also #15, so I'm going to close both issues.

It remains however unclear why I wasn't able to upload a backup via the *Backup and Migrate* restore page. The upload error doesn't seem to be *Backup and Migrate* related, it happens also with language files. Let's explore and discuss it over in the issue [File upload fails: Request Entity Too Large](https://github.com/backdrop-ops/localization/issues/16).
